### DISCUSSION TITLE
Update Slovenian translation from Crowdin

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#8159] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

I have updated to 100% Slovenian translation on crowdin. I kindly ask to proofread all strings and include them in next release. Currently even if they are at 100% they are not included. I suspect proofreading them is missing.

### Solution

Proofread all Slovenian translation string on Crowdin.

### Additional context

Thank you.

**Comment by JC5** [comment]
I will make sure the language is enabled in the layout!

**Comment by thehijacker** [comment]
Language is already available on latest release. But it is incomplete. Even strings that are translated on Crowdin are not included as I am guess they are not approved with proofreading. I have not translated all 100% and also updated few with better translation and I kindly ask to include all my new and updated translations.

**Comment by thehijacker** [comment]
Maybe if you can give me proofreader rights I can approve all the needed and updated strings?

**Comment by JC5** [comment]
I've pulled all new strings on the demo site. Let me know.

**Comment by thehijacker** [comment]
Not all is there. Like this one:

https://crowdin.com/translate/firefly-iii/142/en-sl?filter=basic&value=0#q=premo%C5%BEenjski%20ra%C4%8Duni

I have suggested to have first latter of string upcased. But someone needs to approve this so it is commited. There are also all new strings that I translated but not approved and therefor not used. Please give me proofreading rights and I will go over all not yet approved strings and all strings I suggested better translation and approve them so they are exportable.

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/10611602-8c4d-479b-928a-ca753f9ab4ba)

**Comment by JC5** [comment]
I see what you mean! Making you proofreader would just create more problems for the next person to translate.

Unfortunately, most translations were auto-approved by the system during an import some time ago. So not by me. I have unapproved those strings and your (better) translations should float to the top now.

If you'd wait half a day the demo site should be up-to-date as well.

**Comment by thehijacker** [comment]
Who is the next person that will translate 100% of already translated software? We all want better translation for our native language and I am no different. In last few years I have translated lots of self hosted application to Slovenian and Firefly is no exception. I have already proofreading rights on most of the projects. Among Firefly I have also translated Abacus client (https://github.com/victorbalssa/abacus/issues/186).

Also Slovenian translation was initialy made in 6 years ago and no strings were approved. Could be they were done using some auto translating tools. I again kindly ask for proofreading rights for Slovenian language at least for few days so I can go over all strings again and approved them so they are all correctly exported for next version. After I am finished you can take the proofreading rights away.

**Comment by thehijacker** [comment]
Been spending lots of time on improving translation today. Please let me know if my strings will be added at any time otherwise there is not point on me updating strings on crowdin.

**Comment by JC5** [comment]
Thanks so much! I really appreciate it.

A few years ago I had a similar discussion with a user who then approved AND botched all translations. So I'm a little hesitant to hand over proofreader rights.

In the mean time, I made sure no old strings of mine have been approved. As a result, your efforts should become visible tomorrow.

**Comment by thehijacker** [comment]
For sure I am not here to botch the Slovenian translation, only to improve it and translate all missing string. Which I think it is now almost done. Tested it on demo site and improved couple more strings that I found non translated in between long strings.

But demo page keeps changing to English and it is hard to look over all sections on the page. Is it possible to get the new translations onto my docker Firefly instance? Maybe to replace the existing files somewhere? If I have it on my instance I could take my time to go over everything and see if I missed anything.

**Comment by JC5** [comment]
You can run the `:develop` tag, it is built every night and should already contain your strings.

**Comment by thehijacker** [comment]
Great. Done just that and saw some more non unified translations. Fixing them now and in next few hours. So tomorrom morning they should be applied if I update the :develop docker image?

**Comment by JC5** [comment]
Yes, they should be!

**Comment by thehijacker** [comment]
I noticed some are translated in crowdin but not in :develop tag. Like under OAuth:

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/5c5eaa88-3bae-467d-b6c1-5f5faf77d3af)

I have translated this strings two days ago:

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/5b0dd9a1-91c2-4603-a61d-bf5549f29674)

**Comment by JC5** [comment]
Oh right, I have to rebuild the JS for those. I'll forgot about that.

**Comment by thehijacker** [comment]
So it will be better to go over all menu options tomorrow. Will do. Thank you.

**Comment by thehijacker** [comment]
I have pulled latest develop tag. But again not everything is translated and stays in English even if for that string Slovenian translation exists. When editing transaction for example at the bottom:

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/eca6e57d-d849-4472-97dc-5f2012a43fb7)

Or if I change the target account:

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/f11dc181-9b40-45e3-8bd8-d51b457deaa8)

I refreshed the browser cache. So this is not a browser issue. And it is the same on demo page.

Also I did tons of string updates yesterday evening just to catch the nights build, but they are not all there. This is not an automatic task?

**Comment by JC5** [comment]
Not many people ask for daily updates so no, it's not.

**Comment by thehijacker** [comment]
I will be doing Slovenian translation improvement also today. I kindly ask for you to do full Slovenian language rebuilt over night so I can check everything out. Should be last one as on first strings pass I have everything translated. Just need to go over it again and fix the things I have missed.

My wish is to have it unified and without any typos. Lots of strings to check :).

**Comment by JC5** [comment]
I can't promise anything, but I'll try!

**Reply by thehijacker** [reply]
Any chance for the rebuild on next nightly release? I wish to test the translations. Thank you.

**Comment by thehijacker** [comment]
Looks like it didn't make it to tonights build. Let me know when all files are updated so I can check everything again.

Is there a way for me to download all latest translations and manualy copy them to somewhere in my docker container?

**Comment by thehijacker** [comment]
Looks like tonights build was full and I can see no English strings anymore. I will go over everything in next few days and change to better translation if needed, but I am happy with quick overview. When you will do a release build, please rebuilt Slovenian language in complete so this last minute changes will be applied. Thank you.
